docs/docs/dcap.md

```diff
@@ -54,11 +58,12 @@ You may want to set the Azure DCAP client logging level to FATAL as the default
 export AZDCAP_DEBUG_LOG_LEVEL=FATAL
 ```
 
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
+If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on 
```

Comment:
I realize this is weird somehow actually.
Why should the user manually configure the DCAP plugin if we provide a bundled version? I feel something is odd, why were these instructions written actually @shamsasari ? Should we actually take a look and see what happens if we DO NOT install the Azure DCAP client?

Reply:
I.e. we might actually scrap the installation instructions below, which is quite unnecessary. We might just put a warning about the fact that the Intel libraries should not be there, or otherwise they would override our bundled libs

Reply:
Hi @shamsasari, can you please check this?

Reply:
@bonifazi-r3 you're assuming that the Azure plugin will work in all scenarios, including bare metal outside of Azure's Cloud?

Reply:
I am confused by the instructions even before these changes, also considering that these were written for Graal runtime.
`Otherwise, read on for instructions`.
What is `Otherwise`? I.e. in which case the user code does not actually use the Azure bundle version? The only case I can think of is when the user actually does NOT have any Azure DCAP libraries in `/usr/lib` but the Intel ones. I.e. when the user intentionally does not want to use the Azure DCAP, but they want to use the Intel's one. So in this "Otherwise" paragraph, why are we suggesting the user how to install manually the Azure DCAP client, in this hacky way?

So, my follow up question is: when would the user want to use the Azure `.deb` library from Ubuntu repo instead of using the Azure bundled one? My answer is: **never**, and we should not give further instructions for that.
We have also already clarified that the default libraries are in `/usr/lib` etc, so if we have a hacker user, this would know what to do.

And finally, why would the user need to do that weird manual process, instead of an easy `apt-get install` or similar, depending on the platform?

Reply:
Apologies for the confusion, @JohnJoser3, feel free to setup another chat to discuss.
I would personally do the following: 

- Have an introductory paragraph in relation to DCAP protocol where we do not mentions plugins (basically leave what we have now)
- Then a title like "DCAP client" and describe that it is basically a library `libdcap_quoteprov.so.1` with a symbolic link at `libdcap_quoteprov.so`  that is going to be installed in  in `/usr/lib/` or `/usr/lib/x86_64-linux-gnu` (here I would put something like "apart from the bundled one, see below") and that the users need to be careful to avoid conflicts of plugin and uninstall the ones that they do not want.

Then three subtitles:
 one for the "Intel DCAP client", another for "Azure DCAP client" and the last one one with "Conclave-Azure bundled client".

For the Intel DCAP client (in case of bare metal machines) we need to suggest how to install the package (`sudo apt-get install libsgx-dcap-default-qpl`) but also tell the users that they need to install the PCCS service and just link them the instructions.

For the Azure DCAP client we need to suggest how to install the package, for example with Ubuntu `sudo apt-get install az-dcap-client` and tell the users that they need to be on Azure machines.

As a last subtitle, I would mention that Conclave has a recommended bundled client (an Azure bundled) which is only picked up correctly if no other plugins have been installed in the default directories `/usr/lib/`.
We can, of course, reuse the wording we already have, I feel it is more matter of restructuring and remove the "manual" instructions (the ones that suggest to download and unpack the .deb file), which I don't think are needed.

I am happy to rewrite a proposal so that you can correct, or just let me know how you want to proceed

Reply:
Hi @bonifazi-r3 Marco, I have rewritten the doc according to the instructions. Please take another look.

I have retained the last two sections - `Using Docker container(s)` and `Running a Conclave Application` as these sections have some additional info. Please let me know if I need to remove those too.